apps/web/public/static/locales/en/common.json

```diff
@@ -543,6 +543,7 @@
   "link_shared": "Link shared!",
   "title": "Title",
   "description": "Description",
+  "apps_status": "Apps Status",
```

Comment:
Would "Apps results" make more sense in this context?

Reply:
We need you @Jaibles!